src/Protocol/Protocol_1_8.cpp

```diff
@@ -620,25 +620,19 @@ void cProtocol_1_8_0::SendExperienceOrb(const cExpOrb & a_ExpOrb)
 
 
 
-void cProtocol_1_8_0::SendExplosion(double a_BlockX, double a_BlockY, double a_BlockZ, float a_Radius, const cVector3iArray & a_BlocksAffected, const Vector3d & a_PlayerMotion)
+void cProtocol_1_8_0::SendExplosion(const Vector3f a_Position, const float a_Power)
```

Comment:
Can you remind me why we don't need BlocksAffected anymore?

Reply:
The explosion packet has a field to tell it blocks to set to zero ([ref](https://wiki.vg/Protocol#Explosion)) but since SetBlock already sends this to the client, I wanted to avoid duplication, which should help reduce network traffic. As well avoiding vector allocations in explosion calculation.